ios/FluentUI/Navigation/SideBar.swift

```diff
@@ -0,0 +1,284 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFSideBarDelegate)
+public protocol SideBarDelegate {
+	/// Called after the view representing `TabBarItem` is selected.
+	@objc optional func sideBar(_ sideBar: SideBar, didSelect item: TabBarItem, fromTop: Bool)
+
+	/// Called after the avatar view is tapped in the side bar.
+	@objc optional func sideBar(_ sideBar: SideBar, didActivate avatarView: AvatarView)
+}
+
+@objc(MSFSideBar)
+open class SideBar: UIView {
+	private enum Section: Int {
+		case top
+		case bottom
+	}
+
+	private struct Constants {
+		static let maxTabCount: Int = 5
+		static let viewWidth: CGFloat = 62.0
+		static let avatarViewSize: CGFloat = 30.0
+		static let topItemSpacing: CGFloat = 38.0
+		static let bottomItemSpacing: CGFloat = 28.0
+	}
+
+	@objc public weak var delegate: SideBarDelegate?
+
+	private var avatarViewGestureRecognizer: UITapGestureRecognizer?
```

Comment:
Can we just use a UIButton for more standard accessibility behavior?

Reply:
A UIButton is a view. Avatar view is not a button. We would need to convert AvatarView to a button. Adding an invisible button above the avatar view would be hacky. This is out of scope for this change and I don't even think we want this. Tested accessibility and things work as expected. This is also how we handle the tap events on avatar view in other places in fluentUI (see LargeTitleView).